scalafix-sbt/src/sbt-test/sbt-scalafix/suppress/build.sbt

```diff
@@ -0,0 +1,19 @@
+import _root_.scalafix.Versions
+
+val suppress = project.settings(
+  scalaVersion := Versions.scala212
+)
+
+TaskKey[Unit]("check") := {
+  val s = streams.value
+  val assertContentMatches: ((String, String) => Boolean) =
```

Comment:
Is this necessary? I added it in another file to get IntelliJ support when there were many assertions, but here there is only one assertion.